tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "downlevelIteration": true
```

Comment:
What's this do?

Reply:
Allows spread operator on `Set` objects. I think we need this because we're targeting ES5 not ES6 so it adds some compatibility for older javascript versions. I've added a brief description in the commit message too